### PR TITLE
158 separate informational notifications from warning

### LINKED ISSUE #158: Separate informational notifications from warning
## Description
As a contract manager I want two different information flows so that critical notifications get more attention

## Acceptance criteria
- [x] The informational notifications are separated from the three other types 
- [x] There is a way for the user to see that there are no notifications (e.g. number shown in header, or text shown when empty)

### PR DESCRIPTION
**Pull request by martinjonsson01**
## Summary
Separates informational status updates from the warning status updates on the dashboard.

image

## Definition of done
- [x] All acceptance criteria are completed.
- [x] The code has (at least) 80% branch test coverage.
- [x] All tests pass.
- [ ] The code is reviewed and accepted by (at least) two other team members.
- [x] Views adhere to design style.

Closes #158

### CODE REVIEW

File: src/Client/Pages/Dashboard/StatusUpdates.razor [hunk 1]
```diff
@@ -5,18 +5,39 @@
...
+
+    /// <summary>
+    /// Gets or sets which severities of s<see cref="StatusUpdate"/>s that should be shown.
+    /// </summary>
+    [Parameter]
+    public IEnumerable<AlertLevel> AlertLevels { get; set; } = new[] { AlertLevel.Information, };
+
+    private IEnumerable<StatusUpdate> FilterAlertLevels(IEnumerable<StatusUpdate> statusUpdates)
+    {
+        return statusUpdates.Where(update => AlertLevels.Contains(update.Alert))
```
Comment (JacobBredin) [inline src/Client/Pages/Dashboard/StatusUpdates.razor:40]: Nice use of enums 👍

**Review by JacobBredin — APPROVED** [review]
LGTM! I found a small issue with the lists, they seem to grow indefinitely, maybe you could fix it 😓👉👈 
image

**Comment by martinjonsson01** [comment]
@JacobBredin 

Works on my machine ¯\_(ツ)_/¯
image
Have you tried clearing your cache?

**Comment by JacobBredin** [comment]
try making the page really wide, I know you can do it Mr 32:9

**Comment by martinjonsson01** [comment]
Oh, that's not because of the list styling that's because of the new dashboard. This should've been caught in #179.

**Comment by JacobBredin** [comment]
then it goes in the bug pile 🐛

**Comment by martinjonsson01** [comment]
image

That should do it.

**Review by Matte02 — APPROVED** [review]
LGTM 👍

**Pull request merged**